cds_books/ldap/cli.py

```diff
@@ -81,51 +44,18 @@ def import_users():
 
     click.secho(
         "--- Finished in %s seconds ---" % (time.time() - start_time),
-        fg="green"
+        fg="green",
     )
 
 
 @ldap_users.command(name="sync")
 @with_appcontext
-def sync_users():
-    """Sync ldap with system users command."""
-    import time
-    start_time = time.time()
-
-    ldap_url = current_app.config["CDS_BOOKS_LDAP_URL"]
-    ldap_client = LdapClient(ldap_url)
-    system_users = RemoteAccount.query.join(User).all()
-    ldap_users = ldap_client.get_primary_accounts()
-
-    click.echo("Users fetched")
-
-    ldap_users_map = {}
-
-    for ldap_user in ldap_users:
-        ldap_person_id = ldap_user["employeeID"][0].decode("utf8")
-        ldap_users_map.update({ldap_person_id: ldap_user})
-
-    for system_user in system_users:
-        system_user_person_id = system_user.extra_data["person_id"]
-        ldap_user = ldap_users_map.get(system_user_person_id)
-        if ldap_user:
-            check_user_for_update(system_user, ldap_user)
-        else:
-            click.secho("Deleting user {} with ccid {}".format(
-                system_user.user, system_user_person_id), fg="red")
-            delete_user(system_user)
-
-    # Check if any ldap user is not in our system
-
-    for ldap_user in ldap_users:
-        ldap_mail = ldap_user["mail"][0].decode("utf8")
-        try:
-            User.query.filter(User.email == ldap_mail).one()
-        except NoResultFound:
-            click.secho("Adding new user {}".format(ldap_mail), fg="green")
-            import_ldap_users([ldap_user])
-
-    click.secho(
-        "--- Finished in %s seconds ---" % (time.time() - start_time),
-        fg="green"
-    )
+def ldap_sync_users():
+    """Add entry to database with info about this synchronization."""
+    log = LdapSynchronizationLog.create_cli()
+    result = sync_users()
+    print(result)
```

Comment:
Is this print intended? If yes, why not using the `click.secho()` or a logger?

Reply:
I think it is an old print-debug that shall be removed safely (@KonstantinaStoikou).